boards/nxp/frdm_imx93/doc/index.rst

```diff
@@ -0,0 +1,164 @@
+
+Run following command to test user buttons connected to FRDM-IMX93 GPIO:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/basic/button
+   :host-os: unix
+   :board: frdm_imx93/mimx9352/a55
+   :goals: build
+   :gen-args: -DEXTRA_DTC_OVERLAY_FILE=frdm_imx93_mimx9352_native_btn.overlay
+
+Note: The overlay only supports ``mimx9352/a55``, but can be extended to support
+``mimx9352/m33`` if I2C and PCAL6524 is enabled.
+
+Programming and Debugging (A55)
+*******************************
+
+U-Boot "cpu" command is used to load and kick Zephyr to Cortex-A secondary Core, Currently
+it is supported in : `Real-Time Edge U-Boot`_ (use the branch "uboot_vxxxx.xx-y.y.y,
+xxxx.xx is uboot version and y.y.y is Real-Time Edge Software version, for example
+"uboot_v2023.04-2.9.0" branch is U-Boot v2023.04 used in Real-Time Edge Software release
+v2.9.0), and pre-build images and user guide can be found at `Real-Time Edge Software`_.
+
+.. _Real-Time Edge U-Boot:
+   https://github.com/nxp-real-time-edge-sw/real-time-edge-uboot
+.. _Real-Time Edge Software:
+   https://www.nxp.com/rtedge
+
+Copy the compiled ``zephyr.bin`` to the first FAT partition of the SD card and
+plug the SD card into the board. Power it up and stop the u-boot execution at
+prompt.
+
+Use U-Boot to load and kick zephyr.bin to Cortex-A55 Core1:
+
+.. code-block:: console
+
+    fatload mmc 1:1 0xd0000000 zephyr.bin; dcache flush; icache flush; cpu 1 release 0xd0000000
+
+
+Or use the following command to kick zephyr.bin to Cortex-A55 Core0:
+
+.. code-block:: console
+
+    fatload mmc 1:1 0xd0000000 zephyr.bin; dcache off; icache flush; go 0xd0000000
+
+
+Use this configuration to run basic Zephyr applications and kernel tests,
+for example, with the :zephyr:code-sample:`synchronization` sample:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/synchronization
+   :host-os: unix
+   :board: frdm_imx93/mimx9352/a55
+   :goals: build
+
+This will build an image with the synchronization sample app, boot it and
+display the following console output:
+
+.. code-block:: console
+
+    *** Booting Zephyr OS build v4.1.0-41-g6395333e3d18 ***
+    thread_a: Hello World from cpu 0 on frdm_imx93!
+    thread_b: Hello World from cpu 0 on frdm_imx93!
+    thread_a: Hello World from cpu 0 on frdm_imx93!
+    thread_b: Hello World from cpu 0 on frdm_imx93!
+
+System Reboot (A55)
+===================
+
+Currently i.MX93 only support cold reboot and doesn't support warm reboot.
+Use this configuratiuon to verify cold reboot with :zephyr:code-sample:`shell-module`
+sample:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/subsys/shell/shell_module
+   :host-os: unix
+   :board: frdm_imx93/mimx9352/a55
+   :goals: build
+
+This will build an image with the shell sample app, boot it and execute
+kernel reboot command in shell command line:
+
+.. code-block:: console
+
+    uart:~$ kernel reboot cold
+
+References
+==========
+
+More information can refer to NXP official website:
```

Comment:
this sentence is not grammatically correct.  Also, I think you guys typically include the following -- cc @DerekSnell 

```
.. include:: ../../common/board-footer.rst
   :start-after: nxp-board-footer
```